### PR TITLE
Changes required to publish staging packages

### PR DESCRIPTION
**Pull request by axefinaxe**
Changes required to publish staging packages
They were deployed without the staging tag which clashes with the production package. This adds staging to the tag.

### CODE REVIEW

**Review by lemoninfinex — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -74,5 +74,18 @@ It is possible for Cannon to verify freshly deployed contracts, using the follow
 `./run pnpm cannon verify infinex-multichain:VERSION -c CHAIN_ID --api-key ETHERSCAN_KEY`
 
 
+## useful commands
+ ```
+  CANNON_REGISTRY_PRIORITY=local pnpm cannon alter --chain-id 421614  --subpkg clone.infinex infinex-multichain:0.1.7@O2 mark-complete deploy.Forwarder deploy.AccountFactory invoke.AccountFactoryInitialize deploy.InitialProxyImplementation
+  ```
+   generates an IPFS url that can be upgraded from if steps are executing that shouldn't
+
+   i.e.
```
Comment (rayInfinex) [inline README.md:83]: nit: this should be e.g.

File: README.md [hunk 2]
```diff
@@ -74,5 +74,18 @@ It is possible for Cannon to verify freshly deployed contracts, using the follow
 `./run pnpm cannon verify infinex-multichain:VERSION -c CHAIN_ID --api-key ETHERSCAN_KEY`
 
 
+## useful commands
+ ```
+  CANNON_REGISTRY_PRIORITY=local pnpm cannon alter --chain-id 421614  --subpkg clone.infinex infinex-multichain:0.1.7@O2 mark-complete deploy.Forwarder deploy.AccountFactory invoke.AccountFactoryInitialize deploy.InitialProxyImplementation
+  ```
+   generates an IPFS url that can be upgraded from if steps are executing that shouldn't
```
Comment (rayInfinex) [inline README.md:81]: can you explain when cannon would try to execute steps that it shouldn't be?

File: README.md [hunk 3]
```diff
@@ -74,5 +74,18 @@ It is possible for Cannon to verify freshly deployed contracts, using the follow
...
+
+   i.e. 
+
+```
+ CANNON_REGISTRY_PRIORITY=local pnpm cannon build infinex-multichain/testnet/infinex-multichain-arbitrum-sepolia.toml --chain-id 421614 --private-key  XXXX --upgrade-from @ipfs:QmcU78qLgucSaPerrJqYEaanMCNVZhZbBgSiyGgcyvcpy3
+```
+
+also, when publishing, adding ` --tags 1.0.0` ensures that a package only gets published with the specified space separated tags, and not default which includes latest
```
Comment (rayInfinex) [inline README.md:89]: is this still needed if you have `tags = ["1.0.0staging"]` in the cannonfile?

File: README.md (hunk 2 above)
Comment (axefinaxe) [reply README.md]: adressed

File: README.md (hunk 1 above)
Comment (axefinaxe) [reply README.md]: addressed

**Review by rayInfinex — APPROVED** [review]

**Pull request merged**